Sources/swift-format/Subcommands/LintFormatOptions.swift

```diff
@@ -49,6 +49,9 @@ struct LintFormatOptions: ParsableArguments {
     """)
   var ignoreUnparsableFiles: Bool
 
+  @Flag(help: "Whether or not to run formatting or linting in parallel consuming all possible resources")
```

Comment:
I opted not to put a docstring here since it would say the same thing, happy to add one if we'd like to keep them on everything

Reply:
That's fine, both are probably redundant and the `help` string clearly takes priority since it's user-facing.